Apps/W1/BankDeposits/app/src/permissions/BankDepositsObj.PermissionSet.al

```diff
@@ -3,7 +3,7 @@
 /// do not include this permission set in any other permission set
 /// and do not change the Access and Assignable properties
 /// </summary>
-permissionset 1690 "Bank Deposits - Objects"
+permissionset 1690 "Bank Deposits - Obj."
```

Comment:
The permission set is public, so this would be a breaking change.

The permission set is not assignable so the length should not be a problem.

Reply:
@pri-kise Thanks for reporting this. However as Maria said, fixing this on the application side by renaming the permission set is a breaking change. We'll look into the issue on the platform side.